files/dump/sites/profiles/start.page

```diff
@@ -1,25 +0,0 @@
-Fresh Wikidot Installation
-
-Congratulations, you have successfully installed a new wiki using Wikidot -- a free wiki collaboration software.
-
-+ What to do next
-
-++ Customize this wiki
-
-* Your Wikidot site has two menus, [[[nav:side | one at the side]]] called '{{nav:side}}', and [[[nav:top | one at the top]]] called '{{nav:top}}'.  These are Wikidot pages, and you can edit them like any page.
-* To edit a page, go to the page and click the **Edit** button at the bottom.  You can change everything in the main area of your page.  The Wikidot system is [*http://www.wikidot.org/doc easy to learn and powerful].
-* You can attach images and other files to any page, then display them and link to them in the page.
-* Every Wikidot page has a history of edits, and you can undo anything.  So feel secure, and experiment.
-* If you want to learn more, make sure you visit the [*http://www.wikidot.org/doc Documentation section at www.wikidot.org]
-
-++ Create user accounts
-
-Go to [[[admin:users | Users management panel]]] and create accounts for wiki users.
-
-++ Visit Wikidot.org
-
-Go to **[http://www.wikidot.org www.wikidot.org]** -- home of the Wikidot software -- for extra documentation, howtos, tips and support.
-
-++ Explore more at Wikidot.com
-
-If you want more feature rich wiki reliably hosted, consider creating a wiki within **[http://www.wikidot.com/ Wikidot.com]**
```

Comment:
Do these files already exist in another location? What gets populated for new wikis?

Reply:
These are basically just dumps of stuff that's already in the database. If we want to bootstrap another way of doing the setup down the road that's fine, but I don't want to keep this stuff around for now.